lib/qa_topic_edits.rb

```diff
@@ -155,6 +179,14 @@ def voted
     scope.current_user && ::Topic.voted(object.topic, scope.current_user)
   end
 
+  def vote_count
+    scope.current_user && ::Topic.vote_count(object.topic, scope.current_user)
+  end
+
+  def can_vote
+    scope.current_user && ::Topic.can_vote(object.topic, scope.current_user)
```

Comment:
Let's also get rid of the ``scope.current_user &&`` here too, as there is a user check in the ``can_vote`` method itself.